package.json

```diff
@@ -49,6 +49,7 @@
 		"@types/karma-chai": "^0.1.1",
 		"@types/mocha": "^5.2.5",
 		"@types/sinon": "^7.0.3",
+		"call-timer": "^0.1.0",
```

Comment:
Where is it used?

Reply:
This was used to check the performance of methods like "resize".
Now it isn't necessary at the moment, so I think it's okay to be removed.
It might be needed on later performance enhancement though.